deployment/fetch_vnodes_log.sh

```diff
@@ -14,6 +14,8 @@ Usage(){
     echo "    Optional Arguments:"
     echo "      -b, --ON_BUILD_CONFIG_DIR: The directory of on-build-config. It's required for start"
     echo "      -a, --BMC_ACCOUNT_LIST: The list of bmc account, such as 'admin:admin root:root'. It's required for start"
+    echo "      -n, --NOHUP, start log fetch script in daemon background."
```

Comment:
can you please tell people exmaple in which senario this paramete should be true ?